### PR TITLE
Added compatibility with Node 0.12

### PR DESCRIPTION
**Pull request by ierceg**
Now compatible with 0.12 by using built-in execSync or, for previous versions, using exec-sync as before

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,13 +1,28 @@
...
+  if (typeof ports === 'string') {
+    return ports;
+  }
+
+  const Buffer = require('buffer').Buffer;
+  if (Buffer.isBuffer(ports)) {
+    return ports.toString();
+  }
+  
+  return "[error] Spurious service returned an unsupported result: " + ports;
```
Comment (JakeChampion) [inline src/index.js]: @ierceg What do you think to this throwing an error?

Comment (ierceg) [inline src/index.js]: @JakeChampion I would actually prefer that but I tried to make it work as closely as they way it was before. I'll throw an error here and update the PR.

**Comment by ierceg** [comment]
@JakeChampion Fixed.

**Comment by JakeChampion** [comment]
This is now in the 1.0.1 release. Thanks for the pull request @ierceg!

**Pull request merged**